libcxx/include/__algorithm/simd_utils.h

```diff
@@ -11,7 +11,11 @@
 
 #include <__algorithm/min.h>
 #include <__bit/bit_cast.h>
-#include <__bit/countr.h>
+#if defined(_LIBCPP_BIG_ENDIAN)
+#  include <__bit/countl.h>
+#else
+#  include <__bit/countr.h>
+#endif
```

Comment:
Let's just include both of them unconditionally. They're really not large enough to make it worth the conditional, and we try to avoid including headers conditionally in detail headers.